dal/src/main/java/com/laiteam/echowall/dal/entity/EmailInfo.java

```diff
@@ -0,0 +1,32 @@
+package com.laiteam.echowall.dal.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Component
+public class EmailInfo {
+    private String fromAddress;
+    private String toAddress;
+    private String subject;
+    private String body;
+
+    private static String DEFAULT_SENDER_EMAIL =  "developerforfun2020@gmail.com";
```

Comment:
static final

Reply:
this should be in the application property file.

Reply:
DEFAULT_SENDER_EMAIL is no longer needed